styles/asciidoc.atom-text-editor.less

```diff
@@ -76,6 +76,10 @@ atom-text-editor, :host {
           font-weight: bold;
       }
 
+      &.footnote {
```

Comment:
same thing as #94 , for now try to avoid custom style, use the 'default' codes create a better cross-theme support.

I note we must make a special effort to refactor styling so to have a more homogeneous style.